packages/snowpack-plugin/src/index.ts

```diff
@@ -57,10 +57,7 @@ export default function vanillaExtractPlugin(
       const js = processVanillaFile({
         source,
         filePath,
-        outputCss: !isSSR,
-        identOption:
-          identifiers ??
-          (snowpackConfig.mode === 'production' ? 'short' : 'debug'),
+        outputCss: true,
```

Comment:
It would be great to get this merged!

```suggestion
        outputCss: true,
        identOption:
          identifiers ??
          (snowpackConfig.mode === 'production' ? 'short' : 'debug'),
```